cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -167,7 +167,10 @@ func (w *wizard) doCLIOpt(choice string, options []wizardOpt) {
 	log.Error("That's not something I can do")
 }
 
+var RinkebyNetworkId = "4"
+var DevenvNetworkId = "54321"
+
 func (w *wizard) onRinkeby() bool {
```

Comment:
Not crucial, but perhaps for clarity this could actually be `onTestNet()` since it should return true for any of the test network setups. Up to you if you want to make the tiny change though!

Reply:
Thought about doing exactly that, however for completeness, there would be a few more changes, since we also have `rinkeby` flags in a few places, eg  https://github.com/livepeer/go-livepeer/blob/f5217d7cbe22527de8616bc1d1a03c1242395f54/cmd/livepeer_cli/livepeer_cli.go#L81

Reply:
Fixed. Note that the ETH faucet is now available via devenv CLI, but it's basically a no-op. Enabling it is going to be more work, so leaving it as-is for now.